doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -218,9 +218,6 @@ The following OpenStack components are deployed on the Infrastructure. Some of t
 | (Tool that utilizes APIs)| Heat| the orchestration service| Required| X |  |
 | UI| Horizon| the WEB UI service| Required| X|  |
 | Key Manager| Barbican| the secret data management service| Optional| X |  |
-<!--
```

Comment:
Please do not delete

Reply:
Keep commented the content about Cyborg, as it may be reintroduced if we move to a higher OpenStack release